Trie/Trie/Trie/Trie.swift

```diff
@@ -62,11 +60,32 @@ class Trie {
     fileprivate let root: Node
     fileprivate var wordCount: Int
 
-    /// Creats an empty trie.
-    init() {
+    /// Creates an empty trie.
+    override init() {
```

Comment:
I agree that we shouldn't rely on some special compiler behavior.  I find this type of special behavior annoying.  I can't remember special cases like this!  It's better to do it explicitly.

I made both changes and pushed them.  If there's something else I need to do, let me know.  I sometimes find git overwhelming!